scripts/xtensa-build-all.sh

```diff
@@ -345,9 +379,11 @@ do
 		make overrideconfig
 	fi
 
-	if [ 'tgl' != "${platform}" ]; then
+	# TGL needs MEU arguments for signing
+	if [ 'tgl' == "${platform}" ] && [ -n "${MEU_PATH_OPTION}" ] \
+	   && [ -n "${MEU_OPENSSL_OPTION}" ]; then
```

Comment:
At this late stage in the script execution we should be able to assume that either both `${MEU_ }` variables are defined or none; so the test can be simplified to test only one of them.